mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentMapper.java

```diff
@@ -82,6 +84,7 @@ private Attachment attachment(Row row) {
                 .attachmentId(AttachmentId.from(row.getString(ID)))
                 .bytes(row.getBytes(PAYLOAD).array())
                 .type(row.getString(TYPE))
+                .name(Optional.fromNullable(row.getString(NAME)))
                 .size(row.getLong(SIZE))
```

Comment:
I'm not happy about that, what if we screw up and size becomes != then array.length ?